src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/customer.html.twig

```diff
@@ -49,8 +50,14 @@
             {{ 'View full details'|trans({}, 'Admin.Actions') }}
           </a>
         </div>
+        {% else %}
+        <div class="col">
+          <h2 class="mb-0">{{ 'Deleted customer'|trans({}, 'Admin.Global') }}</h2>
```

Comment:
Could we think about a better workflow when we've to disable margin-bottom ? Using class like mb-0 is against the CSS principle :/

Reply:
I would prefer something like customer-title-delete and adding a margin-bottom: 0; somewhere in the css, or disable margin-bottom on h2 inside a certain class...

What do you think @PierreRambaud

Reply:
Agree with @NeOMakinG It's also easier for someone to do something more specific with a custom class than a class like `mb-0`

Reply:
@PierreRambaud @NeOMakinG Maybe we can open another issue for that because all that page is build like that and I was just reusing a part to adapt it when the customer doesn't exist. WDYT?

Reply:
Ok, but I think we should talk about it with the team before next migrations